src/app/pages/sessions/sessions/sessions.page.ts

```diff
@@ -0,0 +1,64 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+
+@Component({
+  selector: 'app-sessions',
+  templateUrl: './sessions.page.html',
+  styleUrls: ['./sessions.page.scss'],
+})
+export class SessionsPage implements OnInit {
+  type: string;
+  constructor(private activatedRoute: ActivatedRoute) {
+    this.activatedRoute.queryParamMap.subscribe(params => {
+      this.type = params.get('type');
+      if (this.type == null) {
+        this.type = "all-sessions";
+      }
+    });
+  }
+  public headerConfig: any = {
+    menu: false,
+    notification: false,
+    headerColor: 'primary',
+    backButton: true
+  };
+
+  ngOnInit() {
+  }
+  public segmentChanged(ev: any) {
+    if (this.type == "all-sessions") {
```

Comment:
is this condition correct? 

Use ternary opertaor also

Reply:
ok